src/chains/supported.chains.ts

```diff
@@ -1323,7 +1323,7 @@ export const supportedEVMChains: EVMChain[] = [
 export const supportedSolanaChains: SolanaChain[] = [
   {
     key: ChainKey.SOL,
-    chainType: ChainType.Solana,
+    chainType: ChainType.SVM,
```

Comment:
Please don't forget to make changes here https://github.com/lifinance/data-types

Reply:
yes! Will pipeline the changes

Reply:
https://github.com/lifinance/data-types/pull/6

Reply:
Will merge this and release new major as this is breaking changes